### DISCUSSION TITLE
hikaxpro zones color status

### DISCUSSION THREAD

**Discussion by magotecmx** [petrleocompel/hikaxpro_hacs#110] [Q&A] [answered]
I am working with the HACS integration (which is very good by the way) but I have a question. Is there any way in which the states of the zones can be reflected in colors, that is to say, Armed one color, Bypass another color, Triggered another color. I am working on systems with more than 60 zones and I need to display everything on a single dashboard.

**Comment by petrleocompel** [comment, marked as answer]
You can create helpers in HA. 
Or look in to custom dashboard components.

**Reply by magotecmx** [reply]
Thank you, I didn't know about this section. I will develop the solution.
